libraries/radpdfprocessing/formats-and-conversion/convert-to-image/using-image-format-provider.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Using SkiaImageFormatProvider
+page_title: Using SkiaImageFormatProvider
+slug: radpdfprocessing-formats-and-conversion-image-using-skiaimageformatprovider
+tags: pdf,image,skia
+published: True
+position: 1
+---
+
+# Using SkiaImageFormatProvider
+
+Since R3 2022 the __RadPdfProcessing__ library support converting entire documents to images. This is achieved by using the third party library SkiaSharp. You can convert to various formats, using synchronous or asynchronous export. 
+
+>important This feature is only available in the NET Standard version of the suite. For other versions check the fllowing article: [ThumbnailFactory](https://docs.telerik.com/devtools/wpf/controls/radpdfviewer/features/export-fixedpage-to-image)
+>
```

Comment:
Since R3 2022, the __RadPdfProcessing__ library supports converting document pages to images. This is achieved by using the SkiaSharp [add link] library. You can convert to various formats, using synchronous or asynchronous export. 

>important This feature is only available in the NET Standard version of the suite. For other versions, check the following article: [ThumbnailFactory](https://docs.telerik.com/devtools/wpf/controls/radpdfviewer/features/export-fixedpage-to-image).